package.json

```diff
@@ -52,5 +52,8 @@
     "lint-staged": "^13.1.0",
     "semantic-release": "^20.0.2",
     "typescript": "^4.9.4"
+  },
+  "dependencies": {
+    "@metriport/commonwell-sdk": "^1.2.3-alpha.0"
```

Comment:
Curious: why do we need this on the root `package.json`?

Reply:
Must of installed in the wrong folder, updated it